NEWS.md

```diff
@@ -1,9 +1,9 @@
 shinytest (development version)
 ==========
 
-* Also display the massage about where to find the diff when de diff viewer was opened but the diffs were not accepted. ([#131](https://github.com/rstudio/shinytest/issues/131))
+* Also display the message about where to find the diff when the diff viewer was opened but the diffs were not accepted. ([#131](https://github.com/rstudio/shinytest/issues/131))
```

Comment:
`Added a message about....` ?

Reply:
Oh, this news item was for a different PR. I was just fixing typos along the way.